todos/utils.py

```diff
@@ -220,8 +281,13 @@ def fetch_project_with_member_name_start_or_end_with_a():
     Util to fetch project details having members who have name either starting with A or ending with A.
     :return: list of dicts - List of project data
     """
-    # Write your code here
-    pass
+
+    projects = Project.objects.filter(
+        Q(members__first_name__istartswith="A") | Q(members__last_name__iendswith="A")
+    ).distinct()
+
+    serializer = ProjectSerializer(projects, many=True, exclude_fields=["id"])
+    return serializer.data
```

Comment:
```suggestion
    return ProjectSerializer(projects, many=True, exclude_fields=["id"]).data
```

do this at all places possible.